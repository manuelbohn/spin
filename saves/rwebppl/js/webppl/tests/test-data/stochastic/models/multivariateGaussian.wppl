var mu = Vector([-1, 1, 2]);
var cov = Matrix([
  [0.8, 0.4, 0.0],
  [0.4, 1.0, 0.0],
  [0.0, 0.0, 1.2]
]);

var model = function() {
  var x = multivariateGaussian({mu: mu, cov: cov});
  return T.sumreduce(T.range(x, 0, 2));
};
