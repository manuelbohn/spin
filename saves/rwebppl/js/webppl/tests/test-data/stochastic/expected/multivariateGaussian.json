{
  "mean": 0,
  "std": 1.612
}
